dev-haskell/servant-client/servant-client-0.18.3.ebuild

```diff
@@ -1,62 +1,51 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-# ebuild generated by hackport 0.6.6.9999
+# ebuild generated by hackport 0.7.9999
 
 CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
 inherit haskell-cabal
 
 DESCRIPTION="Automatic derivation of querying functions for servant"
-HOMEPAGE="http://docs.servant.dev/"
+HOMEPAGE="https://docs.servant.dev/"
 SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
```

Comment:
I'd probably drop this one, too, as it's more or less equivalent to master's version (but with a dropped test restriction - does this mean the test suite actually works without network access now?).

Reply:
Yes, the tests are working now, for whatever reason